### PR TITLE
fix bugs and add item

### PR DESCRIPTION
**Pull request by Richardsh1**
fix buff item bugs and resubmit

### CODE REVIEW

File: objects/obj_item_parentbuff/Create_0.gml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+item_name = "";
+item_description = "";
+health = 0 ;
```
Comment (Regmyster) [inline objects/obj_item_parentbuff/Create_0.gml:3]: Can we remove the spacing between the semi-colon to keep things consistent?

File: objects/obj_player/Alarm_1.gml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+if (buff_active) {
+    // Buff的持续时间已经结束，取消变化
```
Comment (Regmyster) [inline objects/obj_player/Alarm_1.gml:2]: for consistency and understanding throughout the team, we may want to keep the comments in English.

File: objects/obj_player/Step_0.gml [hunk 3]
```diff
@@ -177,7 +177,56 @@ if (keyboard_check_pressed(ord("E"))) {
 		}
 	}
 }
+//if (bouncing == true && animation_complete) {
```
Comment (Regmyster) [inline objects/obj_player/Step_0.gml:180]: Is there a reason for keeping this commented-out code? if so can we add a comment as to why we are keeping it? can we remove all the extra spacing below the commented-out code to leave just one line of padding?

**Review by Regmyster — COMMENTED** [review]
Things seem good, If you can just go over the few comments left in the review.

File: objects/obj_player/Alarm_1.gml (hunk 2 above)
Comment (Richardsh1) [reply objects/obj_player/Alarm_1.gml]: sorry about it, I forgot to translate into English
Fixed

File: objects/obj_item_parentbuff/Create_0.gml (hunk 1 above)
Comment (Richardsh1) [reply objects/obj_item_parentbuff/Create_0.gml]: Sure
Fixed

File: objects/obj_player/Step_0.gml (hunk 3 above)
Comment (Richardsh1) [reply objects/obj_player/Step_0.gml]: oh the the test code, i forgot to delete
Fixed

**Review by Regmyster — APPROVED** [review]

**Pull request merged**